dvc/repo/experiments/queue/utils.py

```diff
@@ -41,7 +41,7 @@ def fetch_running_exp_from_temp_dir(
         if info.git_url and fetch_refs and info.status > TaskStatus.PREPARING:
 
             def on_diverged(_ref: str, _checkpoint: bool):
-                return False
```

Comment:
During the experiment running, fetch will wrongly make the running experiment as `diverged` without this force operation we can't fetch a fast iterating experiment.